IDEAS/Fluid/HeatExchangers/RadiantSlab/EmbeddedPipe.mo

```diff
@@ -136,6 +136,11 @@ annotation(Dialog(tab="Flow resistance"));
   Modelica.Blocks.Sources.RealExpression[nDiscr] Q_tabs(y=Q)
     annotation (Placement(transformation(extent={{-100,50},{-72,70}})));
 
+  Modelica.Blocks.Math.Sum sumQTabs(nin=nDiscr, k=ones(nDiscr))
+  "Block to sum all the discretizations heat flows"
+    annotation (Placement(transformation(extent={{18,50},{38,70}})));
+  Modelica.Blocks.Interfaces.RealOutput QEmbTotal "Total thermal power of the component going into tabs"
```

Comment:
```suggestion
  Modelica.Blocks.Interfaces.RealOutput QEmbTotal 
    "Total thermal power going into the heat port"
```

Reply:
newline to avoid long lines
the embedded pipe need not always be connected to TABS so I rephrased this